lib/srv/forward/sshserver.go

```diff
@@ -1161,6 +1161,7 @@ func (s *Server) handleAgentForward(ch ssh.Channel, req *ssh.Request, ctx *srv.S
 		if err != nil {
 			return trace.Wrap(err)
 		}
+		ctx.AddCloser(userAgent)
```

Comment:
Can we include test coverage to prevent a regression and make sure this works as intended?

Reply:
+1

Reply:
Done